src/ChunkMap.cpp

```diff
@@ -2593,6 +2593,33 @@ bool cChunkMap::ForEachChunkInRect(int a_MinChunkX, int a_MaxChunkX, int a_MinCh
 
 
 
+void cChunkMap::ForEachLoadedChunk(std::function<bool(int, int)> a_Callback)
+{
+	cCSLock Lock(m_CSLayers);
+	for (cChunkLayerList::const_iterator itr = m_Layers.begin(); itr != m_Layers.end(); ++itr)  // iterate over ALL loaded layers
+	{
+		cChunkLayer * layer = *itr;
+		for (int x = 0; x < LAYER_SIZE; x++)
+		{
+			for (int z = 0; z < LAYER_SIZE; z++)
+			{
+				cChunkPtr p = layer->FindChunk(layer->GetX()*LAYER_SIZE + x, layer->GetZ()*LAYER_SIZE + z);  // A bit stupid to have local->global->local chain...
```

Comment:
Not sure. Adding a function to look up a chunk in local coordinates just for this looks silly. Adding "naked code" for that would mean that if it ever changes there is chance that someone will forget to update this. So no really good way. 
About spaces around operators: can clang style checker check these? Or in general be more exact? It's really annoying when the checker misses them and then waste I other people time.